channels/stable-4.5.yaml

```diff
@@ -37,7 +40,9 @@ versions:
 - 4.5.7
 - 4.5.8
 - 4.5.9
+- 4.5.10
```

Comment:
We're retroactively adding those to stable - is this intended?

Reply:
intended - https://github.com/openshift/cincinnati-graph-data/commit/845ac8263349184aebbbbb3f54a945124f146956